### ISSUE TITLE
Obscure error message on no channel layer

### ISSUE THREAD

**Issue by uhbif19** [datadvance/DjangoChannelsGraphqlWs#17]
When starting Django project without channel layer, I am getting traceback ending like this:

```
  File ".../python3.7/site-packages/channels_graphql_ws/graphql_ws.py", line 1156, in _register_subscription
    waitlist.append(self.channel_layer.group_add(group, self.channel_name))
AttributeError: 'NoneType' object has no attribute 'group_add'
``

This suggests that problem is in `self.channel_layer` equal to None. Adding channel layer to project eleminates error. So, as far as I understand channel layer is required for your libruary.

I think this error is not clear, and for me as novice in Channels is not obvious, that channel layer is required. I suggest to add more clear error message for that case.

**Comment by hozblok** [comment]
Hi @uhbif19,

Indeed, e.g. the `websocket_connect` method of `AsyncWebsocketConsumer` from Channels contains [the check](https://github.com/django/channels/blob/3af5f11190bfd07e29a5d12ce70bb4f138295ed8/channels/generic/websocket.py#L34) with the error message.

So if you have capacity feel free to create a PR that adds validation with a clear error message.

**Comment by prokher** [comment]
Fixed in v0.4.1.
